doc/changelog.md

```diff
@@ -0,0 +1,10 @@
+Change Log
```

Comment:
I think in most open source projects this file is called simply "changes" (generally in caps, but that's not our style) rather than "changelog"